compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -40,7 +40,7 @@ import annotation.tailrec
 import Implicits.*
 import util.Stats.record
 import config.Printers.{gadts, typr}
-import config.Feature, Feature.{migrateTo3, modularity, sourceVersion, warnOnMigration}
+import config.Feature, Feature.{migrateTo3, sourceVersion, warnOnMigration}
```

Comment:
I'd leave modularity as an import. Too much noise to prefix it with Feature everywhere.